### PR TITLE
Add to_base64 function #354

### PR DESCRIPTION
**Pull request by TobiasNx**
See #354

### CODE REVIEW

**Review by blackwinter — CHANGES_REQUESTED** [review]
Also, some (minor) formatting issues.

File: README.md [hunk 1]
```diff
@@ -833,6 +833,16 @@ to_json("<sourceField>"[, pretty: "<boolean>"][, error_string: "<errorValue>"])
 
 [Java Code](https://github.com/search?type=code&q=repo:metafacture/metafacture-fix+path:FixMethod.java+"+to_json+{")
 
+##### `to_base64`
+
+Encodes the value as base64 hash.
```
Comment (blackwinter) [inline README.md:838]:
```suggestion
Replaces the value with its Base64 encoding.
```

File: metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java [hunk 2]
```diff
@@ -4062,4 +4062,26 @@ public void shouldUriEncodePathSegmentWithoutSafeChars() {
...
+                i.startRecord("1");
+                i.startEntity("data");
+                i.literal("title", "this-is-a-test");
+                i.endEntity();
+                i.endRecord();
+            },
+            o -> {
+                o.get().startRecord("1");
+                o.get().startEntity("data");
+                o.get().literal("title", "blabka=");
```
Comment (blackwinter) [inline metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java:4080]: > do you have an idea why the build failes?

In 59ce50c you changed the Base64 value to an incorrect one. I was already wondering what that was about...

**Comment by TobiasNx** [comment]
then closing

**Review by blackwinter — APPROVED** [review]

**Review by fsteeg — APPROVED** [review]

**Pull request merged**